VideoWeb/VideoWeb/Controllers/ParticipantsController.cs

```diff
@@ -299,5 +338,63 @@ public async Task<IActionResult> StaffMemberJoinConferenceAsync(Guid conferenceI
                 return StatusCode(e.StatusCode, e.Response);
             }
         }
+
+        /// <summary>
+        /// This C# function deletes a participant from a conference, handling exceptions and logging
+        /// errors.
+        /// </summary>
+        /// <returns>
+        /// The DeleteParticipantFromConferenceAsync method returns an IActionResult. If the participant
+        /// is successfully removed from the conference, it returns a status code of NoContent (204). If
+        /// there is an error, it may return a status code of NotFound (404) or BadRequest (400)
+        /// depending on the situation. In case of a VideoApiException, it returns the status code and
+        /// response from the exception.
+        /// </returns>
+        [HttpDelete("{conferenceId}/participants/{participantId}")]
+        [SwaggerOperation(OperationId = "DeleteParticipantFromConference")]
+        [ProducesResponseType((int)HttpStatusCode.NoContent)]
+        [ProducesResponseType((int)HttpStatusCode.NotFound)]
+        [ProducesResponseType((int)HttpStatusCode.BadRequest)]
+        public async Task<IActionResult> DeleteParticipantFromConferenceAsync(Guid conferenceId, Guid participantId)
+        {
+            try
+            {
+                await videoApiClient.RemoveParticipantFromConferenceAsync(conferenceId, participantId);
+                string reason = "Participant disconnected removed";
+                await UpdateCacheAndPublishUpdate(conferenceId, reason);
+                return NoContent();
+            }
+            catch (VideoApiException e)
+            {
+                logger.LogError(e, "Unable to delete participant {ParticipantId} from conference {ConferenceId}", participantId, conferenceId);
+                return StatusCode(e.StatusCode, e.Response);
+            }
+        }
+
+        #region Private Methods
```

Comment:
I don't think this region is necessary, it's not consistent with our other controllers

Reply:
removed